guides/common/modules/proc_creating-a-local-source-for-a-custom-file-type-repository.adoc

```diff
@@ -39,7 +39,7 @@ Alternatively, to prevent downtime caused by stopping the service, you can use t
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # {foreman-maintain} packages unlock
-# {project-package-install} python39-pulp_manifest
```

Comment:
This looks like a bug in the docs. Should that be fixed and backported?

Reply:
Yes, we have a Bug report for it: https://issues.redhat.com/browse/SAT-24998